src/Sulu/Bundle/AdminBundle/Resources/js/services/ResourceRequester/ResourceRequester.js

```diff
@@ -9,43 +9,36 @@ const listDefaults = {
     limit: 10,
 };
 
-function buildQueryString(queryOptions: ?Object) {
-    const options = queryOptions;
-    if (!options) {
-        return '';
-    }
+export default class ResourceRequester {
+    static buildQueryString(queryOptions: ?Object) {
```

Comment:
Nice catch! But be aware that the changes I've added in #3658 should also be changed in here.